### PR TITLE
Add example applications for identity propagation with OIDC

### PR DESCRIPTION
**Pull request by fjuma**

### CODE REVIEW

File: oidc-with-identity-propagation/ejb-basic/ejb/src/main/java/org/wildfly/security/examples/ejb_basic/ejb/ManagementBean.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package org.wildfly.security.examples.ejb_basic.ejb;
```
Comment (Skyllarr) [inline oidc-with-identity-propagation/ejb-basic/ejb/src/main/java/org/wildfly/security/examples/ejb_basic/ejb/ManagementBean.java:1]: @fjuma Just a detail, missing copyright here

File: oidc-with-identity-propagation/ejb-basic/ejb/src/main/java/org/wildfly/security/examples/ejb_basic/ejb/Management.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package org.wildfly.security.examples.ejb_basic.ejb;
```
Comment (Skyllarr) [inline oidc-with-identity-propagation/ejb-basic/ejb/src/main/java/org/wildfly/security/examples/ejb_basic/ejb/Management.java:1]: @fjuma Just a detail, missing copyright here

**Review by Skyllarr — COMMENTED** [review]
@fjuma Just checking, should this example have a simple README ?

File: oidc-with-identity-propagation-same-domain/ejb-same-domain/ejb/src/main/java/org/wildfly/security/examples/ejb_same_domain/ejb/WhoAmIBean.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package org.wildfly.security.examples.ejb_same_domain.ejb;
```
Comment (Skyllarr) [inline oidc-with-identity-propagation-same-domain/ejb-same-domain/ejb/src/main/java/org/wildfly/security/examples/ejb_same_domain/ejb/WhoAmIBean.java:1]: @fjuma Just a detail, missing copyright here and above

File: oidc-with-identity-propagation/virtual-security-domain-to-domain/web/src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        <web-resource-collection>
+            <web-resource-name>secured</web-resource-name>
+            <url-pattern>/secured</url-pattern>
+        </web-resource-collection>
+        <auth-constraint>
+            <role-name>*</role-name>
+        </auth-constraint>   
+    </security-constraint>
+
+   <!-- Configure login to be HTTP Basic -->
```
Comment (Skyllarr) [inline oidc-with-identity-propagation/virtual-security-domain-to-domain/web/src/main/webapp/WEB-INF/web.xml:15]: Just a minor, this comment should specify OIDC and not Basic

File: oidc-with-identity-propagation/ejb-basic/ejb/src/main/java/org/wildfly/security/examples/ejb_basic/ejb/ManagementBean.java (hunk 1 above)
Comment (fjuma) [reply oidc-with-identity-propagation/ejb-basic/ejb/src/main/java/org/wildfly/security/examples/ejb_basic/ejb/ManagementBean.java:17]: Thanks, fixed.

File: oidc-with-identity-propagation/ejb-basic/ejb/src/main/java/org/wildfly/security/examples/ejb_basic/ejb/Management.java (hunk 2 above)
Comment (fjuma) [reply oidc-with-identity-propagation/ejb-basic/ejb/src/main/java/org/wildfly/security/examples/ejb_basic/ejb/Management.java:17]: Thanks, fixed.

File: oidc-with-identity-propagation-same-domain/ejb-same-domain/ejb/src/main/java/org/wildfly/security/examples/ejb_same_domain/ejb/WhoAmIBean.java (hunk 3 above)
Comment (fjuma) [reply oidc-with-identity-propagation-same-domain/ejb-same-domain/ejb/src/main/java/org/wildfly/security/examples/ejb_same_domain/ejb/WhoAmIBean.java:17]: Fixed.

File: oidc-with-identity-propagation/virtual-security-domain-to-domain/web/src/main/webapp/WEB-INF/web.xml (hunk 4 above)
Comment (fjuma) [reply oidc-with-identity-propagation/virtual-security-domain-to-domain/web/src/main/webapp/WEB-INF/web.xml]: Good catch, fixed.

**Comment by fjuma** [comment]
> @fjuma Just checking, should this example have a simple README ?

I have a guide in progress so will skip the READMEs for now. All the details will be in the guide.

**Review by Skyllarr — APPROVED** [review]

**Pull request merged**